docs/querying/sql.md

```diff
@@ -168,14 +168,11 @@ There are two important factors that can affect the performance of queries that
 
 ## UNION ALL
 
-The "UNION ALL" operator fuses multiple queries together. Druid SQL supports the UNION ALL operator in two situations:
-top-level and table-level. Queries that use UNION ALL in any other way will not be able to execute.
+The UNION ALL operator fuses multiple queries together. Druid SQL supports the UNION ALL operator in two situations: top-level and table-level, as described below. Queries that use UNION ALL in any other way will not execute.
 
 ### Top-level
 
-UNION ALL can be used at the very top outer layer of a SQL query (not in a subquery, and not in the FROM clause). In
-this case, the underlying queries will be run separately, back to back. Their results will be concatenated together
-and appear one after the other.
+You can use UNION ALL at the very top outer layer of a SQL query - not in a subquery, and not in the FROM clause. The underlying queries run separately, back to back. Their results are concatenated together and appear one after the other.
```

Comment:
```suggestion
You can use UNION ALL at the very top outer layer of a SQL query - not in a subquery, and not in the FROM clause. The underlying queries run sequentially. Druid concatenates their results so that they appear one after the other.
```